Feedback fixes (#12)
lithekod/git-in-git: presentation/main.tex

  Your one stop shop for Game Jams, competitive programming, meetups and other coding related events.

  \vspace{2.0em}
  \textbf{Applications are open for the 21/22 board!} \\
  Mail any questions to \texttt{ordf@lithekod.se} \\
\end{frame}

\begin{frame}[fragile]
  \frametitle{The structure}
  
  \begin{itemize}[<+->]

>     \item Slides are available on \url{www.lithekod.se/git}

Reviewer: ```suggestion
    \item Presentation -- slides at \url{www.lithekod.se/git}
```

    \item Some small exercises \url{www.lithekod.se/git}
    \item We start with the easy stuff
    \item Feel free to ask questions in chat -- answers after the pause
    \item \command{git status} -- a command, can be run in a terminal
  \end{itemize}

\end{frame}

\begin{frame}[fragile]
  \begin{center}

>     I don't need no terminal, I have a GUI

Reviewer: ```suggestion
    I don't need no terminal, I have a GUI!
```

    \command{git clone} & \\
    \command{git init} & \\
  \end{tabular}
  \vspace{1em}

>   Usually given by something like GitHub or GitLab.

Reviewer 2: ```suggestion
  Links usually found at GitHub or GitLab.
```
Denna slide är lite whack för mig 
![image](https://user-images.githubusercontent.com/16568066/106033917-8e326c80-60d2-11eb-88bd-aa490497d378.png)

>   \end{tabular}
>   \vspace{1em}
>   Usually given by something like GitHub or GitLab.

Reviewer: ```suggestion
  \end{tabular} \\
  \vspace{1em}
  Usually given by something like GitHub or GitLab.
```

  Silence.
\end{frame}

\begin{frame}
  \frametitle{Getting to know the family -- git add-ing properly}

  \keyword{Thoughts}
  \begin{itemize}[<+->]

>     \item \dots be specific when adding
>     \item \dots can be hard to ``un-add''
>     \item \dots avoid adding ``everything''
>     \item \dots think twice before adding with wildcards

Reviewer: ```suggestion
    \item Be specific when adding
    \item Can be hard to ``un-add''
    \item Avoid adding ``everything''
    \item Think twice before adding with wildcards
```

\end{verbatim}
\end{frame}

\begin{frame}[fragile]
  \frametitle{Getting to know the family -- push and remotes}
  \keyword{Remote}\\

>   Is a git repo somewhere else. \\

Reviewer: ```suggestion
  A git repo somewhere else. \\
```

  \vspace{1em}

>   If you're standing on a branch noone else knows about, you have

Reviewer 2: ```suggestion
  If you're standing on a branch no one else knows about, you have
```

>   to push the branch aswell\\

Reviewer 2: ```suggestion
  to push the branch as well.\\
```

>   If you're standing on a branch noone else knows about, you have
>   to push the branch aswell\\

Reviewer: ```suggestion
  If you're standing on a branch no one else knows about, you have
  to push the branch as well. \\
```

  \vspace{1em}

>   \command{git push -u origin mybranch} \\

Reviewer 2: ```suggestion
  \command{git push -u origin mybranch} \\
  \vspace{1em}
```

  Pushes the branch \texttt{mybranch} and all the
  commits on it to the server.
  \vspace{1em}

>   Afterwards you can run \command{git push}

Reviewer: ```suggestion
  Afterwards you only have to run \command{git push}.
```

  \frametitle{Branching out -- checkout}

    \keyword{When} \\

>     \command{git checkout} - The sonic screwdriver of Git. Gets stuff from somewhere else. \\[1em]

Reviewer: ```suggestion
    \command{git checkout} - The TARDIS of Git. Can take you anywhere, anytime. \\[1em]
```

>     \command{git checkout} - The sonic screwdriver of Git. Gets stuff from somewhere else. \\[1em]

Reviewer 2: ```suggestion
    \command{git checkout} - The sonic screwdriver of Git. Used mainly to jump between branches. \\[1em]
```

    Silence.
\end{frame}

\begin{frame}
  \frametitle{Branching out -- creating a branch}

  \command{git branch newbranch} \\
  \command{git checkout newbranch} \\
  Creates and moves to the new branch.
  \vspace{1em}

>   \command{git cbeckout -b newbranch} \\

Reviewer: ```suggestion
  \command{git checkout -b newbranch} \\
```

>   Combines both commands, used by speedrunners.

Reviewer: ```suggestion
  Combines both commands. Used by speed runners.
```

>   Combines both commands, used by speedrunners.

Reviewer 2: ```suggestion
  \command{git checkout -b newbranch} \\
  Combines both commands, used by speedrunners.
```
Very epic

    This may also result in \alert{merge conflicts}. \\[1em]
\end{frame}

\begin{frame}[fragile]

>   \frametitle{Basic commans -- Summary}

Reviewer: ```suggestion
  \frametitle{Basic commands -- Summary}
```

  \begin{tabular}{ll}
    \keyword{Basic} & \\
    \command{git add} & I want to keep these! \\
    \command{git commit} & I did a thing! \\
    \command{git push} & I want to share this! \\ % mention pull

>     \keyword{Informative} & \\

Reviewer 2: Tror inte keyword behövs här

Reviewer 2: Förtydligande: Jag tror det är bättre att bara ha listan.

Author: Jag tycker den gör det lättare att se vad som är vad. Mycket lättare att ta till sig information när det finns struktur. Så här skulle det seut.

![image](https://user-images.githubusercontent.com/19999497/106036401-98099f00-60d5-11eb-8d5e-94acc963b727.png)

Reviewer 2: Det kanske är bättre att ha den som den är

    \command{git status} & What is happening? \\
    \command{git diff} & What has changed? \\
    \command{git log} & What has happened? \\
    \command{git show} & What changed over there? \\

    \keyword{Create branch} & \\
    \command{git checkout} & Jump to a branch! \\
    \command{git branch} & Manage branches! \\
    \command{git log} & Find commits to jump to! \\

    \keyword{Combine branch} & \\
    \command{git merge} & I want both this and that! \\

>     \command{git pull} & I want \emph{your} stuff! \\

Reviewer: Den här listan är precis lite för lång för vår frame-storlek. Finns det något sätt att göra den mindre? Till exempel tror jag inte vi behöver ha med `log` två gånger.

  \end{tabular}
\end{frame}

> \section{Pause}

Reviewer: Jag tycker vi har en section "Pause // Q/A" istället för en Pause och en Q/A. Det blir lite väl mycket outline annars.

\begin{frame}[fragile]
  \begin{center}

>     Everyone take 5!

Reviewer: ```suggestion
    Everyone take a short break!
```

  \end{center}
\end{frame}

\section{Q/A}

\begin{frame}[fragile]
  \begin{center}

>     Questions!

Reviewer: ```suggestion
    Questions?
```

  \end{center}
\end{frame}

\section{Working with others}
\subsection{Creating your first Merge Request}

\begin{frame}[fragile]
  \frametitle{Project management -- Hosting}
  \vspace{1em}

  GitLab and GitHub are popular hosting sites for projects.

>   They're both just Git-repos on another computer.

Reviewer: ```suggestion
  They're both Git-repositories on someone else's computer.
```

  \vspace{1em}

>   LiU supplies a GitLab instance, and GitHub offers a student version.

Reviewer: ```suggestion
  LiU supplies a GitLab instance, and GitHub offers free GitHub Pro for students.
```

\end{frame}

\begin{frame}[fragile]

>   \frametitle{Project management -- Continous Integration} % not always needed etc

Reviewer: ```suggestion
  \frametitle{Project management -- Continuous Integration} % not always needed etc
```
